cadcutils/cadcutils/util/utils.py

```diff
@@ -76,19 +76,30 @@
 from operator import attrgetter
 import hashlib
 import os
-from distutils.version import StrictVersion
+from packaging.version import Version
 import requests
+import time
+import json
+import warnings
+from pathlib import Path
+from cadcutils import exceptions
 
-__all__ = ['IVOA_DATE_FORMAT', 'date2ivoa', 'str2ivoa',
-           'get_logger', 'get_log_level', 'get_base_parser', 'Md5File']
+__all__ = ['IVOA_DATE_FORMAT', 'date2ivoa', 'str2ivoa', 'get_url_content',
+           'get_logger', 'get_log_level', 'get_base_parser', 'Md5File',
+           'check_version', 'VersionWarning']
 
 # TODO both these are very bad, implement more sensibly
 IVOA_DATE_FORMAT = "%Y-%m-%dT%H:%M:%S.%f"
 
+CADC_CACHE_DIR = os.path.join(os.path.expanduser("~"), '.config')
+VERSION_REFRESH_INTERVAL = 24 * 60 * 60  # 24h
```

Comment:
Is the `CACHE_REFRESH_INTERVAL` meant to be a different value, with different context, than the `VERSION_REFRESH_INTERVAL`?

Reply:
I've changed the name `CACHE_REFRESH_INTERVAL` -> `REG_REFRESH_INTERVAL` to avoid confusion and show that they are meant to be different. `REG` is more critical and volatile and therefore meeds to be refreshed more often.